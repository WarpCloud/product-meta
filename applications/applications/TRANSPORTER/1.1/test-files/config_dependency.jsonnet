{
  application_type: "TRANSPORTER",
  application_name: "transporter1",
  application_version: "5.0",
  user_config: {},
}
