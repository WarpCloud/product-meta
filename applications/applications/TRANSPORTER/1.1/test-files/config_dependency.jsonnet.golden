{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "transporter1",
            "application_type": "TRANSPORTER"
         },
         "configs": {
            "tdt_cpu_limit": 1,
            "tdt_cpu_request": 0.10000000000000001,
            "tdt_memory_limit": 4,
            "tdt_memory_request": 1,
            "tdt_tmp_storage": {
               "accessMode": "ReadWriteOnce",
               "limits": {
                  "blkio.throttle.read_iops_device": 0,
                  "blkio.throttle.write_iops_device": 0
               },
               "size": "256Gi",
               "storageClass": "silver"
            }
         },
         "dependencies": [
            {
               "moduleName": "hdfs",
               "name": "hdfs"
            },
            {
               "moduleName": "inceptor",
               "name": "inceptor"
            },
            {
               "moduleName": "txsql",
               "name": "txsql"
            }
         ],
         "instance_settings": { },
         "moduleName": "tdt",
         "name": "transporter1-tdt",
         "version": "5.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
